library/std/arithmetic.qs

```diff
@@ -289,4 +316,167 @@ namespace Microsoft.Quantum.Arithmetic {
         }
     }
 
+    //
+    //
+    //      New arithmetic operations starts here.
+    // Once it is done, previous implementation will be removed.
+    //
+    //
+
+    //
+    // Operation: Add      |    Ripple-carry    | Carry look-ahead
+    // ____________________________________________________________
+    // y += 5              |  RippleCarryIncByL✔|              N/A
+    // y += x              | RippleCarryIncByLE✔| LookAheadIncByLE
+    // z = x + 5 (z was 0) |                N/A |              N/A
+    // z = x + y (z was 0) |   RippleCarryAddLE✔|   LookAheadAddLE
+    // z += x + 5          |                N/A |              N/A
+    // z += x + y          |                N/A |              N/A
+    //
+
+    /// # Summary
+    /// Increment a little-endian ys register by a BigInt number c
+    ///
+    /// # Description
+    /// Compute ys += c modulo 2ⁿ, where ys is a little-endian register
+    /// of length n > 0, c >= 0 is a BigInt constant, and c < 2ⁿ.
+    /// This operation can be used when the choice of implementation
+    /// is not important.
+    @Config(Full)
+    operation IncByL (c : BigInt, ys : Qubit[]) : Unit is Adj + Ctl {
+        RippleCarryIncByL(c, ys);
+    }
+
+    /// # Summary
+    /// Increments a little-endian register ys by a little-endian register xs
+    ///
+    /// # Description
+    /// Compute ys += xs modulo 2ⁿ, where xs and ys are little-endian registers,
+    /// n = Length(ys), and Length(xs) <= Length(ys).
+    /// This operation can be used when the choice of implementation
+    /// is not important.
+    @Config(Full)
+    operation IncByLE (xs : Qubit[], ys : Qubit[]) : Unit is Adj + Ctl {
+        RippleCarryIncByLE(xs, ys);
+    }
+
+    /// # Summary
+    /// Set a zero-initialized little-endian register zs to the sum of
```

Comment:
```suggestion
    /// Sets a zero-initialized little-endian register zs to the sum of
```

Reply:
Rather than describing what a function does, I actually prefer the imperative form with simple present tense - same form when we give instructions and directions. Many of our functions are commented that way. So when you look at your program you have something like this:
- Allocate qubit
- Apply H gate
- Measure result
- Print the result
rather than description of what each function does.
We have a mix, so I update a few other places.

Reply:
I checked and most places use this 'descriptive' form rather than imperative form. I'll update a few place to use s-es.

Reply:
I should have commented on consistency. It's also okay to change the other occurrences.